model-optimizer/extensions/front/AttributedRollToRoll.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from extensions.ops.roll import Roll
+from mo.front.common.replacement import FrontReplacementPattern
+from mo.graph.graph import Graph, rename_nodes
+from mo.ops.const import Const
+
+
+class AttributedRollToRoll(FrontReplacementPattern):
+    """
+    This transformation converts AttributedRoll operation (axes and shift are specified as attributes) to Roll
+    operation (Inference Engine semantic).
+    """
+    enabled = True
+
+    def find_and_replace_pattern(self, graph: Graph):
+        for attr_roll in graph.get_op_nodes(op='AttributedRoll'):
+            original_name = attr_roll.soft_get('name', attr_roll.id)
+
+            new_roll = Roll(graph, {'axes': attr_roll.soft_get('axes', None),
+                                    'shift': attr_roll.soft_get('shift', None)}).create_node()
```

Comment:
Why you copy attributes `axes` and `shift` of `AttributedRoll` into `Roll`?

Reply:
I used AttributedPadToPad as example and attributes are copied there. But I think in Roll it is not necessary. I will remove the coping.